cicd/jenkins/jenkins-agent/Dockerfile

```diff
@@ -1,4 +1,7 @@
+FROM node:latest AS node_base
```

Comment:
Could we pin this to a specific version? (number 6 https://snyk.io/blog/10-docker-image-security-best-practices/)